lib/application/constants/strings.dart

```diff
@@ -65,6 +65,8 @@ const executionAnswer = 'Resposta';
 const executionNext = 'Próxima';
 const executionCheckAnswer = 'Ver resposta';
 
+const executionDiscard = 'Descartar';
+const executionBackToStudy = 'Voltar ao estudo';
```

Comment:
```suggestion
const executionBackToStudy = 'Voltar ao Estudo';
```

Following the same as the destructive action text